src/components/NavMobile/NavMobile.scss

```diff
@@ -0,0 +1,198 @@
+@import '../../styles/fonts.scss';
+@import '../../styles/utilities/breakpoints.scss';
+
+.nav-mobile {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  overflow: hidden;
+  height: 5rem;
+  padding: 1rem 2rem;
+  @media (min-width: $breakpoint-laptop) {
+    display: none;
+  }
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__logo {
+    width: 7rem;
+    height: 7rem;
+    margin-left: -.5rem;
+    fill: $darkest-purple;
+  }
+
+  &__title {
+    margin: 0;
+    padding: 0;
+    color: $dark-grey;
+    font-size: 1.5rem;
+
+    &--sm {
+      margin: 0;
+      padding: 0;
+      color: $black;
+      font-size: .8rem;
+    }
+  }
+
+  &__sidebar {
+    position: fixed;
+    top: 0;
+    right: -35rem;
+    z-index: 999;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    width: 35%;
+    height: 100%;
+    padding: 2rem;
+    background-color: $white;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
+    &--active {
+      @keyframes slideIn {
+        from {
+          right: -35rem;
+        }
+  
+        to {
+          right: 0;
+        }
+      }
+
+      @extend .nav-mobile__sidebar;
+      position: absolute;
+      top: 0;
+      right: 0;
+      animation: slideIn 200ms ease-out;
+    }
+
+    &--inactive {
+      @keyframes slideOut {
+        from {
+          right: 0rem;
+        }
+  
+        to {
+          right: -35rem;
+        }
+      }
+
+      @extend .nav-mobile__sidebar;
+      position: absolute;
+      top: 0;
+      right: -35rem;
+      animation: slideOut 200ms ease-out;
+    }
+
+    &--close {
+      align-self: flex-end;
+      margin: 1rem 0;
+    }
+
+    &--items {
+      display: flex;
+      flex-direction: column;
+      margin: 0;
+      padding: 0;
+      list-style-type: none;
+    }
+
+    &--item {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: space-between;
+      padding: 1rem 0;
+      font-weight: 600;
+    }
+
+    &--carrot {
+      @keyframes derotateCarrot {
+        0%   {
+          transform: rotate(180deg);
+        }
+
+        100% {
+          transform: rotate(0deg);
+        }
+      }
+      
+      animation: derotateCarrot 200ms ease-in-out forwards;
+
+      &--active {
+        @keyframes rotateCarrot {
+          0%   {
+            transform: rotate(0deg);
+          }
+
+          100% {
+            transform: rotate(180deg);
+          }
+        }
+
+        animation: rotateCarrot 200ms ease-in-out forwards;
+      }
+    }
+
+    &--dropdown{
+      &--active {
+        @keyframes dropDown {
+          0% {
+            transform: scaleY(0)
+          }
+
+          80% {
+              transform: scaleY(1.1)
+          }
+
+          100% {
+              transform: scaleY(1)
+          }
+        }
```

Comment:
Lets move this to bottom of file or to its own keyframes.scss file - we shouldnt nest keyframes like this